tests/acceptance/features/apiContract/accessFileByFileId.feature

```diff
@@ -0,0 +1,110 @@
+@api
```

Comment:
how about this name --> `apiContract/getFileByFileId.feature`

Reply:
I will be adding the tests related to `updating files using file id` in the same file so `getFileByFildId.feature` might not be a good name.

Reply:
I'm not sure if we should add this test in `apiContract`.
We are not doing actual contract testing in `apiContract` however it's about checking the response fields of the different methods and expecting the response fields to be correct. I don't think this test is doing that? maybe we can add this somewhere else?

cc @saw-jan

Reply:
The scenarios added here should not be considered as contract tests.

Reply:
So where should I add this feature file? any suggestions @SwikritiT @saw-jan

Reply:
is there's gonna be more tests then maybe new folder and test cases by fileID can be added there